src/lib/grammar.js

```diff
@@ -80,6 +80,7 @@ const GRAMMAR = {
     'IMPORTANT'     : '!',
     // https://regex101.com/r/mM2vT9/8
     'NAMED'         : '([\\w$]+(?:(?:-(?!\\-))?\\w*)*)',
+    'CSS_VARIABLE'  : '(--[\\w-]+)',
```

Comment:
Can't use `\w` in a character match `[]`

Reply:
```suggestion
    'CSS_VARIABLE'  : '(--[a-zA-Z0-9_-]+)',
```

Reply:
I assume `_` is valid for custom properties but maybe worth double-checking?

Reply:
Clarified: `\w` is valid in char selector, and is only escaped here because the regex is defined as a string to be used in a `new Regex()` constructor